web_src/js/features/comp/SearchRepoBox.ts

```diff
@@ -0,0 +1,31 @@
+import {fomanticQuery} from '../../modules/fomantic/base.ts';
+
+const {appSubUrl} = window.config;
+
+export function initCompSearchRepoBox() {
+  // on the page "page-content organization teams"
+  const $searchRepoBox = fomanticQuery('#search-repo-box');
+  const showFullName = $searchRepoBox.data('full-name');
+  $searchRepoBox.search({
+    minCharacters: 2,
+    apiSettings: {
+      url: `${appSubUrl}/repo/search?q={query}&uid=${$searchRepoBox.data('uid')}`,
+      onResponse(response: any) {
+        const items = [];
+        for (const item of response.data) {
+          let title = item.repository.full_name.split('/')[1];
+          if (showFullName) {
+            title = item.repository.full_name;
+          }
+          items.push({
+            title,
+            description: item.repository.full_name,
```

Comment:
It's not right.

If you have "show full name", then why duplicate the description with the full name again.

Reply:
538536d6f095947c31334ea62eb3860c12373579